src/ucs/sys/topo.c

```diff
@@ -11,6 +11,40 @@
 #include <ucs/sys/topo.h>
 #include <ucs/type/status.h>
 #include <stdio.h>
+#include <ucs/datastruct/khash.h>
+#include <ucs/type/spinlock.h>
+#include <ucs/debug/log.h>
+#include <ucs/debug/assert.h>
+
+#define UCS_TOPO_MAX_SYS_DEVICES 1024
+
+typedef int64_t ucs_bus_id_bit_rep_t;
+
+typedef enum ucs_topo_kh_put {
+    UCS_KH_PUT_FAILED = -1,
+    UCS_KH_PUT_KEY_PRESENT,
+    UCS_KH_PUT_BUCKET_EMPTY,
+    UCS_KH_PUT_BUCKET_CLEAR
+} ucs_topo_kh_put_t;
```

Comment:
need to move this to khash.h

Reply:
`khash.h` seems untouched from the version here: https://github.com/attractivechaos/klib/blob/master/khash.h
Should we be adding a new file called `khash_util.h`, move the above there and include that in `khash.h`?

Reply:
no need, khash.h already includes some changes